### ISSUE TITLE
What is the file CompanyList?

### ISSUE THREAD

**Issue by eddiejaoude** [ugglr/Remote-Junior-Developer-jobs-directory#2]
- What is the file for?
- What is the extension?
- Naming is consistent with other files

**Comment by ugglr** [comment]
Hmm, I think you are right to leave it out of there before there was anything in there, now it's a piece of rubbish lurking there. 

I'm thinking about putting real coding challenges there sorted by company, the ones they send out to juniors to test their coding skills. It's very hard to measure your skills in a useful way, in that way juniors can see what's expected of them. 

It could be framework sorted as well, or language sorted, or It could be timed challenges with a limit ("you should be able to code this in about 1h") type questions- or just simple apps with normal user stories or other functionality.

**Comment by eddiejaoude** [comment]
> you should be able to code this in about 1h

I found giving someone a test that is not achievable in the time is best, because then can see where they prioritise their efforts 🤓 - just my 2 cents

**Comment by ugglr** [comment]
Yes, I think that is fine, it makes sense from the recruiting side. 

It should be fine to state in the test specification for instance, "hey: it's not about finishing this test in the time given, the whole point is to showcase where you prioritize your efforts in that amount of time". 

There are not that many companies willing to give juniors a chance, it might be that after months of efforts of applying only one or two might be willing to give them a shot. If they are then given a timed test with a workload way beyond and they don't know this way of thinking they might just hit the ground running and start coding and not showcasing their real potential. If done a bad job, they won't be able to go back and apply in a while if a new position comes out. 

But if they know these sort of tests exist they might be able to stay calm and then focus on the architecture or write it in pseudo code, and lastly start to implement the functions of there is time left.

I truly do not like tests that have a very short time limit, it's stressful and stressful thinking is not a good representation. There's also an element of time stealing when the tech does not work, or the internet connection drops, or the in-browser IDE is slow, etc.   

I would rather have an open time limit, and then discuss with the recruiter why I took the time I did. For instance, this was totally new to me and I had to learn my way through, or similar. 

TLDR: Let's get everybody on the same level of understanding and take some stress away, it's stressful enough to apply and get rejected for x months straight.

**Comment by ugglr** [comment]
I'm revamping the site so this is not relevant anymore.
